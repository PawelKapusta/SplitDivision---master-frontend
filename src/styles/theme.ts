import { DefaultTheme } from "styled-components";

export const theme: DefaultTheme = {
    palette: {
        white: "#fff",
        black: "#000",
        background1: "#0F1624",
        gold: "#eaaf57",
    },
    breakpoints: {
        sm: "screen and (max-width: 640px)",
        md: "screen and (max-width: 768px)",
        lg: "screen and (max-width: 1024px)",
        xl: "screen and (max-width: 1280px)",
    },
};
