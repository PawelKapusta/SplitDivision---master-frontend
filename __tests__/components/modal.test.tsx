import { render } from "@testing-library/react";
import Modal from "@components/modal/index";
import { mockTheme } from "../mocks/theme";
import { ThemeProvider } from "styled-components";

describe("modal component", () => {
    it("renders correctly", () => {
        const { container } = render(
            <ThemeProvider theme={mockTheme}>
                <Modal
                    isOpen={false}
                    onClose={() => {
                        return true;
                    }}
                >
                    <div>Children</div>
                </Modal>
            </ThemeProvider>,
        );
        expect(container).toMatchSnapshot();
    });
});
