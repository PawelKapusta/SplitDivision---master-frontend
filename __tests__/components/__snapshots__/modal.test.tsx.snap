// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal component renders correctly 1`] = `
<div>
  <div
    class="sc-51f8e2a0-0 kBmVQJ"
  >
    <div
      class="sc-51f8e2a0-1 YlaYB"
    >
      <div>
        Children
      </div>
      <button>
        <img
          alt="Close-icon.svg"
          data-nimg="1"
          decoding="async"
          height="35"
          loading="lazy"
          src="/icons/close-icon.svg"
          style="color: transparent;"
          width="35"
        />
      </button>
    </div>
  </div>
</div>
`;
